marimo/_output/formatters/holoviews_formatters.py

```diff
@@ -37,23 +36,7 @@ def _show_chart(
                 | hv.core.ndmapping.NdMapping
             ),
         ) -> tuple[KnownMimeType, str]:
-            backend_output = hv.render(plot)
-
-            # If its a dict, then its a plotly figure,
-            # and we should convert it to a plotly object
-            if DependencyManager.plotly.has() and isinstance(
-                backend_output, dict
-            ):
-                plotly_html = PlotlyFormatter.render_plotly_dict(
-                    backend_output
-                )
-                return ("text/html", plotly_html.text)
-
-            # Call as_html to recurse back into the formatter
-            # this may be bokeh, matplotlib, or plotly
-            html = as_html(backend_output)
-
-            return ("text/html", html.text)
+            return from_panel(plot)._mime_()
```

Comment:
is it possible for holoviews be installed, but not panel?

Reply:
It is not.